admin/app/views/appConfig.scala.html

```diff
@@ -0,0 +1,75 @@
+@(message: String = "")(implicit request: RequestHeader, context: model.ApplicationContext)
+
+@admin_main("App Config", isAuthed = true) {
+
+    <h1>App Config</h1>
+
+    <h3>Information:</h3>
+    Configuration is loaded by each app from parameter store. Properties are loaded from the following levels:
+    <ol>
+        <li>Local filesystem overrides</li>
+        <li>/frontend/{stage}/{app}/{config.property.name}</li>
+        <li>/frontend/{stage}/{config.property.name}</li>
+        <li>/frontend/{config.property.name}</li>
+    </ol>
+
+    Properties can override each other. <br/>
+    Parameter store configuration can be modified via the <a href="https://eu-west-1.console.aws.amazon.com/ec2/v2/home?region=eu-west-1#Parameters:sort=Name">AWS console</a>. <br/>
+
+
+    <h3>Find parameters</h3>
+        Searching for properties via the console is difficult - you must supply a full prefix including the config path.<br/>
+        The following tool provides a parameter store substring search for frontend properties existing in:
+    <ol>
+        <li>/frontend/{stage}/{app}/{config.property.name}</li>
+        <li>/frontend/{stage}/{config.property.name}</li>
+        <li>/frontend/{config.property.name}</li>
+    </ol>
+
+    <hr/>
+
+    <style>
+
+    </style>
+
+    <form method="GET" id="findForm">
+        <input class="img-url form-control" name="property" type="text" value="" placeholder="config.property.name"/>
+        <button class="btn btn-default" type="submit">Find</button>
+    </form>
+    <br>
+    <div>
+        <b>Status:</b> <span id="status">Ready</span> <br/>
+        <b>Results:</b>
+        <br/>
+        <div class="key-value" id="key-values">
+            key : value
+        </div>
+        <br/>
+        <br/>
+    </div>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <script>
+        $("#findForm").submit(function(event) {
+            $("#key-values").empty();
+            $("#status").text("Finding config...");
+
+            event.preventDefault();
+
+            var $form = $(this),
+                term = $form.find('input[name="property"]').val(),
+                url = "/config/parameter/" + term;
+
+            var eventGetParams = $.get(url);
+
+            eventGetParams.done(function(data) {
+                $("#status").text("Ready");
+                Object.keys(data).forEach(function(key) {
+                    $("#key-values").append("<p>"+key+" <b>:</b> "+data[key]+"</p>");
```

Comment:
Do we want to show the `value`. There is some security implication. Personnally I won't show it, or maybe only the first 2 chars

Reply:
Okay, I'll make the change to the endpoint